scripts/package

```diff
@@ -13,16 +13,67 @@
 
 cp -r bin package/
 
-APIVERSION=`./bin/longhorn-share-manager version --client-only|jq ".clientVersion.apiVersion"`
+#APIVERSION=`./bin/longhorn-share-manager version --client-only|jq ".clientVersion.apiVersion"`
+
+# Try bin/longhorn-share-manager-amd64 and bin/longhorn-share-manager-arm64 doesn't have "Exec format error"
```

Comment:
As you use the right arch image to get the version already below, the comment here should be not required anymore.

Reply:
Forgot to remove it. Thank you.